app/views/namespaces/_namespace.html.slim

```diff
@@ -1,6 +1,7 @@
 tr id="namespace_#{namespace.id}"
   td= link_to namespace.clean_name, namespace
   td= namespace.repositories.count
+  td= namespace.webhooks.count
```

Comment:
I just wonder, wouldn't it better to have a link leading to the page where all the webhooks are shown? Would that be possible? If yes I think we can deal with that by opening an issue and making a dedicated PR. This one is already pretty big.